CompetitionTemplate/src/competition/subsystems/drive/DriveSubsystem.java

```diff
@@ -66,38 +70,39 @@ public void tankDrive(double leftPower, double rightPower) {
     
     public void tankDriveSafely(double leftPower, double rightPower) {
         
-        
-        // if we are incredibly tipped over, don't bother, we can't save ourselves. Also disable safeties,
-        // they will be re-enabled once we right ourselves.
-        if (Math.abs(pose.getRobotPitch()) > 90) {
-            tippedRecently = true;
-            leftPower = 0;
-            rightPower = 0;
-        }
-        // if we are pitching a lot, AND tip prevention is enabled, AND we haven't fully tipped over recently,
-        // try and fix the situation using tipPower.
-        else if (Math.abs(pose.getRobotPitch()) > 30 
-                && tipPreventionEnabled.get() == true 
-                && tippedRecently == false) {
-            double tipPower = tipPreventionPower.get();
-            if (pose.getRobotPitch() > 0) {            
-                leftPower = -tipPower;
-                rightPower = -tipPower;
+        if (enableSafeTankDrive.get()) {
+            // if we are incredibly tipped over, don't bother, we can't save ourselves. Also disable safeties,
+            // they will be re-enabled once we right ourselves.
+            if (Math.abs(pose.getRobotPitch()) > 90) {
```

Comment:
maybe latch to log when we enter one of these states so we can see after a match why things went crazy?